### DISCUSSION TITLE
Error on a 64-bit host (a pi5) when trying to customize 32-bit images

### DISCUSSION THREAD

**Discussion by tinker2much** [gitbls/sdm#204] [General]
I'm re-testing some of my configuring and burning scripts after a bit of a hiatus. Things changed: I upgraded to 11.8, and I'm trying this on a pi5 (I used to run on a pi4). 

I typically make 4 base images - 64-bit full, 64-bit, 32-bit, and 32-bit lite. I noticed the following error message on the two 32-bit ones.   
> * Phase 0 Completed
> * Enter image '/home/pi/Downloads/bookworm32.img' for Phase 1
> /bin/bash: error while loading shared libraries: libtinfo.so.6: ELF load command address/offset not page-aligned
> ?! systemd-nspawn failed on this IMG
>    Host system architecture: aarch64 with 16384 byte pages
>    IMG binaries: ELF 32-bit LSB executable
>    Correct this error on this host by:
>     * sudoedit /boot/firmware/config.txt
>     * Add the following line:
>     * kernel=kernel8.img
>     * Reboot and rerun the sdm command
>    Details: https://github.com/raspberrypi/bookworm-feedback/issues/120

I can't switch to kernel8, given that the pi5 rootfs is encrypted. I had run into the issue of being unable to --explore 32-bit images on the pi5; I should have realized that there might also be problems with _creating_ them. (Slow to get the memo...)

Could you please confirm that I can't process 32-bit images on the pi5? If so, I'll probably just revert to doing them all on a pi4, since splitting up the scripts and working on two machines would be a bother.

**Comment by gitbls** [comment]
Confirmed for Bookworm 32-bit images. https://github.com/gitbls/sdm/issues/123 and see the linked RaspberryPi issue for more details.

Note that it does work for Buster and Bullseye 32-bit images. Don't know offhand how they are linked, but apparently not the same as Bookworm 32-bit.
